nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1066,6 +1066,13 @@ def fields_validator(cls, fields) -> Self:
         return fields
 
 
+class FullResourceApplyTo(BaseModel):
```

Comment:
What about naming this more generically, just in case? `ApplyTo`

Reply:
I prefixed it with FullResource on purpose so if we need a different ApplyTo for a different RAG strategy they don't collide. But I can change it if you don't like it